OData/src/System.Web.OData/OData/Builder/ODataConventionModelBuilder.cs

```diff
@@ -622,6 +623,12 @@ private void MapEntityType(EntityTypeConfiguration entity)
                             addedNavigationProperty.Contained();
                         }
 
+                        AutoExpandedAttribute autoExpandedAttribute = property.GetCustomAttribute<AutoExpandedAttribute>();
```

Comment:
Why add these codes here?

You insert the convention into the list, it should be apply automatically.